src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -159,4 +164,33 @@ export class Feedback implements BaseEntity {
             feedback.type = FeedbackType.AUTOMATIC_ADAPTED;
         }
     }
+
+    /**
+     * Helper method to build the feedback text for the review. When the feedback has a link with grading instruction
+     * it merges the feedback of the grading instruction with the feedback text provided by the assessor. Otherwise,
+     * it return detail_text or text property of the feedback depending on the submission element.
+     *
+     * @param feedback that contains feedback text and grading instruction
+     * @returns {string} formatted string representing the feedback text ready to display
+     */
+    public static buildFeedbackTextForReview(feedback: Feedback): string {
```

Comment:
why is it a static method? why not make it a getter or a regular method?

Reply:
well, no specific reason, it can be a regular method.

Reply:
actually, its not needed. I changed it. @Olimbia23  could you also have a look the Apollon side, when you have a time?